packages/server/logging/index.ts

```diff
@@ -24,14 +24,14 @@ export default function (app: express.Express) {
       register: prometheusClient.register,
       collectionPeriodMilliseconds: highFrequencyMetricsCollectionPeriodMs(),
       config: {
-        knex
+        getDbClients: getAllRegisteredDbClients
```

Comment:
you could get all clients from the config and then this would not rely on any DB state.